task/fossa-scanner/0.1/test/run.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: fossa-scan-springboot
+spec:
+  workspaces:
+    - name: source
+      persistentVolumeClaim:
+        claimName: my-source-repo
+  taskRef:
+    name: fossa-scanner
+  podTemplate:
+    securityContext:
+      runAsUser: 0
```

Comment:
any reason for running the tests as root?

Reply:
used for test, will remove this